scripts/tasks/babel.ts

```diff
@@ -0,0 +1,45 @@
+import { transformAsync } from '@babel/core';
+import * as glob from 'glob';
+import fs from 'fs';
+import path from 'path';
+import { EOL } from 'os';
+
+const OEL_REGEX = new RegExp(EOL, 'g');
+
+function addSourceMappingUrl(code: string, loc: string): string {
+  return code + '\n//# sourceMappingURL=' + path.basename(loc);
+}
+
+export async function babel() {
+  const files = glob.sync('{lib,lib-commonjs}/**/*.js');
+
+  for (const filename of files) {
+    const filePath = path.resolve(process.cwd(), filename);
+
+    const codeBuffer = await fs.promises.readFile(filePath);
```

Comment:
nit, you can use `fs-extra` for a simpler API (optional though)

Reply:
Did not get it. Can you please provide a suggestion what we can simplify there with `fs-extra` usage there? 🙂

Reply:
What I was thinking is that since fs-extra returns promises by default, you could just do `fs.readFile` (instead of `fs.promises.readFile`). But thinking about it more, it's not really important.